query_optimizer/tests/ExecutionHeuristics_unittest.cpp

```diff
@@ -75,12 +75,12 @@ class ExecutionHeuristicsTest : public ::testing::Test {
     std::vector<attribute_id> build_attribute_ids;
     build_attribute_ids.push_back(build_attribute_id);
     QueryPlan::DAGNodeIndex build_operator_index =
-        query_plan->addRelationalOperator(new BuildHashOperator(*build_relation,
+        query_plan->addRelationalOperator(new BuildHashOperator(0,  /* dummy query ID */
```

Comment:
This suggestion will require some work as there are a number of unit tests which need this change.  The dummy query ID flag is informative as of now. We can come back to it later, if that's fine.